core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/recipe/RecipeEngine.java

```diff
@@ -111,7 +111,7 @@ public void executePostAmbariStartRecipes(Stack stack, Set<HostGroup> hostGroups
 
     public void executePostInstallRecipes(Stack stack, Set<HostGroup> hostGroups) throws CloudbreakException {
         Orchestrator orchestrator = stack.getOrchestrator();
-        if (shouldExecuteRecipeOnStack(hostGroups, orchestrator, POST_CLUSTER_INSTALL)) {
+        if (recipesSupportedOnOrchestrator(orchestrator)) {
```

Comment:
I think we need method name like: defaultRecipesAreSupportedOnOrchestrator in order to avoid the same problem happen again

Reply:
otherwise the reader of the code will never know there are default recipes in the PostInstall phase in some cases